pennylane/qchem/observable_hf.py

```diff
@@ -135,13 +136,17 @@ def qubit_observable(o_ferm, cutoff=1.0e-12, mapping="jordan_wigner"):
     """
     if mapping == "jordan_wigner":
         h = qml.jordan_wigner(o_ferm, ps=True, tol=cutoff)
+
     elif mapping == "parity":
         qubits = len(o_ferm.wires)
         h = qml.parity_transform(o_ferm, qubits, ps=True, tol=cutoff)
     elif mapping == "bravyi_kitaev":
         qubits = len(o_ferm.wires)
         h = qml.bravyi_kitaev(o_ferm, qubits, ps=True, tol=cutoff)
 
+    if list(h.wires) != sorted(list(h.wires)):
+        h = sum(i * h[i] for i in sorted(h.keys(), key=lambda x: list(x.wires)))
```

Comment:
This sorts the Hamiltonian into always ascending wire order?

Reply:
Yes! In almost all cases, the Hamiltonian naturally has an ascending wire order but there are a few edge cases.

Reply:
Should there be a mention of this as a bug fix in the changelog?

Reply:
This is not really a bug, the old and new Hamiltonians are both correct but the new one is more consistent with the rest of qchem.